shared/android/settings.gradle

```diff
@@ -0,0 +1,33 @@
+rootProject.name = 'Keybase'
+
+apply from: '../node_modules/react-native-unimodules/gradle.groovy'
+includeUnimodulesProjects([modulesPaths: ['../../node_modules']])
```

Comment:
still `../..`?

Reply:
yeah, we actually don't need it because that's what it defaults to